shared/actions/chat.js

```diff
@@ -831,8 +831,9 @@ function _unboxedToMessage (message: MessageUnboxed, idx: number, yourName, your
 
       switch (payload.messageBody.messageType) {
         case CommonMessageType.text:
-          // If we get a histocal message w/ messageID and outboxID ignore the outboxID
-          const outboxID = (isHistory && common.messageID) ? undefined : payload.clientHeader.outboxID && outboxIDToKey(payload.clientHeader.outboxID)
+          // If we get a historical message w/ messageID and outboxID ignore the outboxID
+          const messageIsYours = common.author === yourName
```

Comment:
lets chat more about this tomorrow. i think we never want the outbox ids ever if we can